packages/plugin-kafka/src/rdkafka-extra-types.ts

```diff
@@ -1,16 +1,17 @@
-import { Client, Producer, KafkaConsumer, ProducerStream, ConsumerStream } from 'node-rdkafka'
+import { EventEmitter } from 'events'
 import { ConnectOptions } from './types'
 
 declare module 'node-rdkafka' {
-  interface ProducerStream extends Writable {
+  interface ProducerStream extends WritableStream {
```

Comment:
@AVVS  Is it ok? if:
https://github.com/Blizzard/node-rdkafka/blob/master/index.d.ts#L1
https://github.com/Blizzard/node-rdkafka/blob/master/index.d.ts#L241